src/NXP/MathExecutor.php

```diff
@@ -415,13 +427,9 @@ public function getVar(string $variable)
      * @param  string $variable
      * @param  integer|float $value
      * @return MathExecutor
-     * @throws MathExecutorException
      */
     public function setVar(string $variable, $value) : self
     {
-        if (!is_numeric($value)) {
-            throw new MathExecutorException("Variable ({$variable}) value must be a number ({$value}) type ({gettype($value)})");
-        }
         $this->variables[$variable] = $value;
```

Comment:
Wondering if we should just allow int, float, string and bool.  Other types don't make any sense, but for sure string is good.  Might as well add bool too.  Should eval to 0 and 1, so that would seem to work.  Right now this allows any type, so not good.

Reply:
Done using `is_scalar`